gateway-server/src/test/java/org/apache/knox/gateway/services/security/impl/JettySSLServiceTest.java

```diff
@@ -333,13 +339,14 @@ public void TestBuildSslContextFactoryExplicitTrustStoreErrorGettingPassword() t
     String identityKeyAlias = "server";
     Path truststorePath = Paths.get(basedir, "target", "test-classes", "keystores", "server-truststore.jks");
     String truststoreType = "jks";
+    String truststorePasswordAlias = "trust_store_password";
 
-    GatewayConfig config = createGatewayConfig(true, true, identityKeystorePath, identityKeystoreType, identityKeyAlias, truststorePath, truststoreType);
+    GatewayConfig config = createGatewayConfig(true, true, identityKeystorePath, identityKeystoreType, identityKeyAlias, truststorePath, truststoreType, truststorePasswordAlias);
 
     AliasService aliasService = createMock(AliasService.class);
     expect(aliasService.getGatewayIdentityKeystorePassword()).andReturn(identityKeystorePassword).atLeastOnce();
     expect(aliasService.getGatewayIdentityPassphrase()).andReturn(identityKeyPassphrase).atLeastOnce();
-    expect(aliasService.getPasswordFromAliasForGateway(eq("gateway-truststore-password"))).andThrow(new AliasServiceException(null)).atLeastOnce();
+    expect(aliasService.getPasswordFromAliasForGateway(eq("trust_store_password"))).andThrow(new AliasServiceException(null)).atLeastOnce();
```

Comment:
I think this can be truststorePasswordAlias instead of the string?

Reply:
Fixing...